schemas/Account.json

```diff
@@ -31,6 +31,10 @@
     "is_admin": {
       "description": "admin flag",
       "type": "boolean"
+    },
+    "disabled": {
+      "description": "Admin users may temporarily disable an account by setting this flag",
```

Comment:
I meant in your quotation there is a typo. `disabled` is sufficiently clear, but you're right that `is_disabled` is more consistent with `is_admin`. I guess we should either go with `is_disabled` or rename `is_admin` to `admin`.